src/Lykke.Job.TxDetector/Sagas/ConfirmationsSaga.cs

```diff
@@ -18,23 +25,34 @@ public class ConfirmationsSaga
         private readonly ILog _log;
         private readonly IClientSettingsRepository _clientSettingsRepository;
         private readonly IClientAccountsRepository _clientAccountsRepository;
+        private readonly IAssetsServiceWithCache _assetsService;
+        private readonly IAppGlobalSettingsRepositry _appGlobalSettingsRepositry;
+        private readonly IBalanceChangeTransactionsRepository _balanceChangeTransactionsRepository;
+        private readonly IInternalOperationsRepository _internalOperationsRepository;
 
         public ConfirmationsSaga(
             [NotNull] ILog log,
             [NotNull] IClientSettingsRepository clientSettingsRepository,
-            [NotNull] IClientAccountsRepository clientAccountsRepository)
+            [NotNull] IClientAccountsRepository clientAccountsRepository,
+            [NotNull] IAssetsServiceWithCache assetsService,
+            [NotNull] IAppGlobalSettingsRepositry appGlobalSettingsRepositry,
+            [NotNull] IBalanceChangeTransactionsRepository balanceChangeTransactionsRepository,
+            [NotNull] IInternalOperationsRepository internalOperationsRepository)
         {
-            _log = log ?? throw new ArgumentNullException(nameof(log));
+            _log = log.CreateComponentScope(nameof(ConfirmationsSaga));
             _clientAccountsRepository = clientAccountsRepository ?? throw new ArgumentNullException(nameof(clientAccountsRepository));
+            _assetsService = assetsService ?? throw new ArgumentNullException(nameof(assetsService));
+            _appGlobalSettingsRepositry = appGlobalSettingsRepositry ?? throw new ArgumentNullException(nameof(appGlobalSettingsRepositry));
+            _balanceChangeTransactionsRepository = balanceChangeTransactionsRepository ?? throw new ArgumentNullException(nameof(balanceChangeTransactionsRepository));
+            _internalOperationsRepository = internalOperationsRepository ?? throw new ArgumentNullException(nameof(internalOperationsRepository));
             _clientSettingsRepository = clientSettingsRepository ?? throw new ArgumentNullException(nameof(clientSettingsRepository));
         }
 
+        [Obsolete("Method is not deleted now only for compatibility purpose. Should be deleted after next release.")]
         private async Task Handle(CashInOperationCreatedEvent evt, ICommandSender sender)
         {
-            await _log.WriteInfoAsync(nameof(ConfirmationsSaga), nameof(CashInOperationCreatedEvent), evt.ToJson(), "");
-
-            ChaosKitty.Meow();
-
+            _log.WriteInfo(nameof(CashInOperationCreatedEvent), evt, "");
```

Comment:
maybe add some Msg ... "Handle received message". It's can help for found all log by handle